index.html

```diff
@@ -25,8 +25,143 @@
                 </ul>
             </nav>
         </div>
+    </header>
+
+    <main>
+        <section class="board">
+            <h1 class="board__title">LIME PUBLISHING <i>bringing you stories to warm your heart and </i></h1>
+            <h3 class="board__text">
+                <div class="board__text board__text-1">Lorem ipsum dolor sit amet consectetur adipisicing elit. A, necessitatibus quo impedit omnis voluptate non laudantium vero. Sed facere hic dicta omnis aliquam dolore. Totam voluptas doloremque quasi delectus sed?
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. A, necessitatibus quo impedit omnis voluptate non laudantium vero. Sed facere hic dicta omnis aliquam dolore. Totam voluptas doloremque quasi delectus sed?
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. A, necessitatibus quo impedit omnis voluptate non laudantium vero. Sed facere hic dicta omnis aliquam dolore. Totam voluptas doloremque quasi delectus sed?
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                </div>
+                    
+                <div class="board__text board__text-2">Lorem ipsum dolor sit amet consectetur adipisicing elit. A, necessitatibus quo impedit omnis voluptate non laudantium vero. Sed facere hic dicta omnis aliquam dolore. Totam voluptas doloremque quasi delectus sed?
```

Comment:
I think it's easier to read and edit if you instead write:
<div class>
<p class> text </p>
</div> 
for each of these, and remove the h3 tag surrounding it :)